### PR TITLE
LRDOCS-7532 wordsmithing Site Building per feedback

### PR DESCRIPTION
**Pull request by mwilliams2014**
@jrhoun here are the updates for Site Building finally :-) let me know if anything else is needed. Thanks! Supersedes https://github.com/jrhoun/liferay-learn/pull/87

### CODE REVIEW

File: docs/dxp/7.x/en/site-building/building-sites/building-sites-with-site-templates.md [hunk 1]
```diff
@@ -41,8 +45,8 @@ To create a site using a Site Template, follow these steps:
        To view a newly created *blank site*, you must first create a page for it. See `Adding a Page to a Site <../creating-pages/adding-pages/adding-a-page-to-a-site.md>`_ for more information.
     ```
 
-## Additional Information
+## Related Information
 
 * [Introduction to Site Building](../introduction-to-site-building.md)
-* Creating a Site Template
+* [Creating a Site Template](./building-sites-with-site-templates.md)
```
Comment (jrhoun) [inline docs/dxp/7.x/en/site-building/building-sites/building-sites-with-site-templates.md:51]: @mwilliams2014 was this intended to point to itself?

Comment (mwilliams2014) [reply docs/dxp/7.x/en/site-building/building-sites/building-sites-with-site-templates.md:51]: Definitely not. That should point to a different article

File: docs/dxp/7.x/en/site-building/introduction-to-site-building.md [hunk 2]
```diff
@@ -1,3 +1,40 @@
 # Introduction to Site Building
```
Comment (jrhoun) [inline docs/dxp/7.x/en/site-building/introduction-to-site-building.md:1]: @mwilliams2014 i think this article is missing the images referenced. Can you add them to this pull?

Comment (jrhoun) [reply docs/dxp/7.x/en/site-building/introduction-to-site-building.md:1]: nm, they were hiding somewhere else. :)

**Comment by jrhoun** [comment]
Merged. Thanks! Most changes were to implement better cross referencing between existing articles and the ones you've added @mwilliams2014 . I also updated the links in the introduction, added it to the left nav, and refactored out a couple redundant `README.rst` files.

**Comment by mwilliams2014** [comment]
Thanks @jrhoun !

**Pull request closed without merging**